.github/actions/release-maven/action.yaml

```diff
@@ -65,18 +65,18 @@ runs:
     - name: Push changes to new branch
       shell: bash
       run: |
-        git checkout -b ${{ github.ref_name }}-version-bump
-        git push --force origin ${{ github.ref_name }}-version-bump
+        git checkout -b ${{ github.module }}-version-bump
+        git push --force origin ${{ github.module }}-version-bump
     - name: Create pull request
       uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
       with:
         script: |
           const { repo, owner } = context.repo;
           const pullResult = await github.rest.pulls.create({
-            title: 'chore: bump release version ${{ github.ref_name }}',
+            title: 'chore: bump release version ${{ github.module }}',
             owner,
             repo,
-            head: '${{ github.ref_name }}-version-bump',
+            head: '${{ github.module }}-version-bump',
```

Comment:
@devtobi  has problemes.

https://github.com/it-at-m/refarch/actions/runs/13309294214/job/37167532946 


the github.module doesnt exist in the github context https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#github-context